src/components/invoices/SelectionResultsContainer.js

```diff
@@ -30,13 +30,17 @@ const contractsToTooltip = (orgUnit, period) => {
 const SelectionResultsContainer = (props) => {
   const { classes, t, orgUnits, levels, period, pager } = props;
   const invoices = PluginRegistry.extension("invoices.invoices");
+  const dataEntries = PluginRegistry.extension("dataentry.dataEntries");
   const filteredOrgunits = [];
   const omitedOrgunits = [];
 
   if (orgUnits)
     for (let ou of orgUnits) {
       const codes = invoices.getInvoiceTypeCodes(ou, period);
-      if (codes && codes.length > 0) {
+      const activeContract = ou.activeContracts && ou.activeContracts[0];
```

Comment:
1. this looks really specific to burundi
2. this code is in project that don't have contracts module enabled
3. not all projects have data entries enabled too